demos/smart_classroom_demo/include/smart_classroom_demo.hpp

```diff
@@ -184,6 +215,38 @@ DEFINE_double(exp_r_fd, 1.15, face_threshold_output_message);
 /// It is an optional parameter
 DEFINE_int32(last_frame, -1, last_frame_message);
 
+/// @brief Label of teacher<br>
+/// It is an optional parameter
+DEFINE_string(teacher_id, "", teacher_id_message);
+
+/// @brief Min action duration<br>
+/// It is an optional parameter
+DEFINE_double(min_ad, 1.0, min_action_duration_message);
+
+/// @brief Same action time delta<br>
+/// It is an optional parameter
+DEFINE_double(d_ad, 1.0, same_action_time_delta_message);
+
+/// @brief Labels of student actions<br>
+/// It is an optional parameter
+DEFINE_string(student_ac, "sitting,standing,raising_hand", student_actions_message);
+
+/// @brief Labels of student actions for top-k mode<br>
+/// It is an optional parameter
+DEFINE_string(top_ac, "sitting,raising_hand", top_actions_message);
+
+/// @brief Labels of teacher actions<br>
+/// It is an optional parameter
+DEFINE_string(teacher_ac, "standing,writing,demonstrating", teacher_actions_message);
+
+/// @brief Define target action name for top-k mode <br>
+/// It is an optional parameter
+DEFINE_string(top_id, "", target_action_index_message);
```

Comment:
```suggestion
DEFINE_string(top_id, "raising_hand", target_action_index_message);
```

Reply:
I think it is a good idea if the default action is "raising hand" w.r.t. the default actions in `top_ac`.    
(I hope nobody is going to detect the first sitting student)

Reply:
As far as I understand, empty parameter `top_id` means that `TOP_K` mode is turned off.

May be it is better to use the parameter `-a_top` for that?
Make the parameter `-a_top` by default have value `0` or `-1`, and write in help a comment that "if this parameter is positive, the demo detects first K persons with the action, pointed by the parameter 'top_id'" -- ?

Reply:
Added